build/Dockerfile.nginx

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1.4
+FROM nginx:1.25.1-alpine
+
+ARG NJS_DIR
+ARG NGINX_CONF_DIR
+
+RUN apk add --no-cache libcap \
+    && mkdir -p /var/lib/nginx /usr/lib/nginx/modules \
+    && setcap 'cap_net_bind_service=+eip' /usr/sbin/nginx \
```

Comment:
I don't think we need `i` here. I believe in NIC it is there because in NIC the NIC process launches NGINX master.